server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/EmailSetSerializer.scala

```diff
@@ -260,7 +262,117 @@ class EmailSetSerializer @Inject()(messageIdFactory: MessageId.Factory, mailboxI
     readMapEntry[ClientPartId, ClientEmailBodyValue](s => Id.validate(s).fold(e => Left(e.getMessage), partId => Right(ClientPartId(partId))),
       clientEmailBodyValueReads)
 
-  private implicit val emailCreationRequestReads: Reads[EmailCreationRequest] = Json.reads[EmailCreationRequest]
+  case class EmailCreationRequestWithoutHeaders(mailboxIds: MailboxIds,
```

Comment:
Wouldn't it be better to move that one in `EmailSet` scala file instead?

Reply:
It is only used for deserialization (and avoiding the JsonPath pattern) hence I located it here...